bundles/org.openhab.binding.adorne/README.md

```diff
@@ -0,0 +1,96 @@
+# Adorne Binding 
+
+The Adorne Binding integrates [Adorne Wi-Fi ready devices](https://www.legrand.us/adorne/products/wireless-whole-house-lighting-controls.aspx) (switches, dimmers, outlets) from [Legrand](legrand.com).
+
+Legrand attempted to provide a public API based on Samsung's ARTIK Cloud and the initial version of this binding was based on that API. However, Samsung shut down ARTIK Cloud shortly after the release and Legrand has not offered a public API replacement since. That leaves direct interaction with the Adorne Hub as the only control option. Consequently the OpenHAB server and the Adorne Hub must be located on the same network.
+
+The Adorne Hub supports a REST API, but unfortunately there is no documentation or official support from Legrand. This binding's implementation of the REST API is motivated by the great work of [sbozarth](https://github.com/sbozarth/homebridge-lc7001) who figured out the API details.
+
+## Supported Things
+
+| Thing Type|Description|
+|-|-|
+|hub|The Adorne [Hub LC7001](https://www.legrand.us/adorne/products/wireless-whole-house-lighting-controls/lc7001.aspx) serves as the bridge to control all Adorne devices|
+|switch|All Adorne switches and outlets|
+|dimmer|All Adorne dimmers|
+
+## Discovery
+
+Auto-discovery is supported as long as the hub can be discovered using the default host and port. If the hub requires custom host and/or port configuration then setup through a .things file is needed. 
```

Comment:
Applied

Reply:
Since the PaperUI discovery process doesn't allow user supplied parameters I can't discover via PaperUI if host and port are needed to discover the hub.